README.md

```diff
@@ -6,16 +6,19 @@ A mighty, modern CSS linter and fixer that helps you avoid errors and enforce co
 
 ## Features
 
--   **Over one hundred and fifty rules:** Including those that:
-    -   **Catch errors**: e.g. invalid hex colors, duplicate selectors and overriding shorthand properties.
-    -   **Limit language features**: e.g. vendor prefixes, patterns for selectors and specific units, properties, functions and at-rules.
-    -   **Enforce stylistic conventions**: e.g. the whitespace around the colon in declarations and the number of empty lines between rules.
--   **Support for the latest CSS syntax:** Including custom properties, calc() and nesting.
--   **Understands *CSS-like* syntaxes:** The linter is powered by [PostCSS](https://github.com/postcss/postcss), so it understands any syntax that PostCSS can parse, including SCSS, [SugarSS](https://github.com/postcss/sugarss), and Less.
+-   **Over one hundred and fifty built-in rules.** Including ones that:
```

Comment:
“one hundred and ~fifty~ sixty”. Currently, there are 161 rules, and 2 more coming.